docs/source/getting-started/index.rst

```diff
@@ -0,0 +1,96 @@
+.. Copyright (c) 2020 The DAML Authors. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _new-quickstart:
+
+Full Stack DAML
+###############
+
+**Disclaimer:** This guide is being actively developed.
+Expect major changes to the tutorial text and minor changes to the template application.
+
+The goal of this tutorial is to get you up and running with full-stack, DAML-driven app development. We will provide you with the template for a miniature social networking app, get you writing your first end-to-end feature and finally deploy your new app to a persistent ledger.
+
+By the end of the tutorial, you should have an idea of what DAML contracts and ledgers are, how the UI interacts with them, and how you might solve a potential use case with a DAML-driven solution. We do not aim to give a comprehensive guide to all DAML concepts and tools; for that, see the later sections of the documentation. With that, let's get started!
```

Comment:
IMO, we should have a very brief description of the app we're building before people need to download anything. Just make it clear we're doing something cool and it's totally worth downloading all these tools.